library/Fission/Internal/Text.hs

```diff
@@ -0,0 +1,15 @@
+module Fission.Internal.Text
+  ( split256
+  ) where
+
+import Fission.Prelude
+import RIO.NonEmpty
+import qualified RIO.Text as Text
+
+split256 :: Text -> NonEmpty Text
+split256 txt = snd $ Text.foldr splitter (0, ("" :| [])) txt
```

Comment:
https://www.stackage.org/haddock/lts-15.5/rio-0.1.14.1/RIO-Text.html#v:chunksOf

Reply:
```haskell
foo :: Text -> Maybe (NonEmpty Text)
foo txt = nonEmpty $ Text.chunksOf 256 txt

-- or

import qualified RIO.NonEmpty.Partial as Partial.NonEmpty

bar :: Text -> NonEmpty Text
bar txt = Partial.NonEmpty.fromList $ Text.chunksOf 256 txt
```

I'd probably do the former, since it's safer. Also safer than the original version.

Reply:
beautiful